README.md

```diff
@@ -6,7 +6,7 @@
 EPICS is a ruby implementation of the [EBICS](https://www.ebics.org/) (Electronic Banking Internet
 Communication Standard).
 
-It supports EBICS 2.5.
+It supports EBICS 2.4, 2.5 and 3.0.
```

Comment:
I think the readme could reflect which one is the default when initializing a new or existing client and how to work with another version